crates/sui-types/src/temporary_store.rs

```diff
@@ -1303,22 +1305,32 @@ impl<S: GetModule + ObjectStore + BackingPackageStore> TemporaryStore<S> {
         }
     }
 
-    /// Check that this transaction neither creates nor destroys SUI. This should hold for all txes except
-    /// the epoch change tx, which mints staking rewards equal to the gas fees burned in the previous epoch.
-    /// Specifically, this checks two key invariants about storage fees and storage rebate:
-    /// 1. all SUI in storage rebate fields of input objects should flow either to the transaction storage rebate, or the transaction non-refundable storage rebate
-    /// 2. all SUI charged for storage should flow into the storage rebate field of some output object
+    /// Check that this transaction neither creates nor destroys SUI. This should hold for all txes
+    /// except the epoch change tx, which mints staking rewards equal to the gas fees burned in the
+    /// previous epoch.  Specifically, this checks two key invariants about storage fees and storage
+    /// rebate:
+    ///
+    /// 1. all SUI in storage rebate fields of input objects should flow either to the transaction
+    ///    storage rebate, or the transaction non-refundable storage rebate
+    /// 2. all SUI charged for storage should flow into the storage rebate field of some output
+    ///    object
+    ///
     /// If `do_expensive_checks` is true, this will also check a third invariant:
-    /// 3. all SUI in input objects (including coins etc in the Move part of an object) should flow either to an output object, or be burned as part of computation fees or non-refundable storage rebate
-    /// This function is intended to be called *after* we have charged for gas + applied the storage rebate to the gas object,
-    /// but *before* we have updated object versions.
-    /// if `do_expensive_checks` is false, this function will only check conservation of object storage rea
-    /// `epoch_fees` and `epoch_rebates` are only set for advance epoch transactions.
-    /// The advance epoch transaction would mint `epoch_fees` amount of SUI, and burn
-    /// `epoch_rebates` amount of SUI. We need these information for conservation check.
+    ///
+    /// 3. all SUI in input objects (including coins etc in the Move part of an object) should flow
+    ///    either to an output object, or be burned as part of computation fees or non-refundable
+    ///    storage rebate
+    ///
+    /// This function is intended to be called *after* we have charged for gas + applied the storage
+    /// rebate to the gas object, but *before* we have updated object versions.  If
+    /// `do_expensive_checks` is false, this function will only check conservation of object storage
+    /// rea `epoch_fees` and `epoch_rebates` are only set for advance epoch transactions.  The
```

Comment:
"rea"?

Reply:
I wish we did not change this comment and left it alone.... lol....

Reply:
This is my fault -- I just couldn't read it how it was before. I couldn't figure out what "rea" meant before, my guess is "rebates", but I didn't want to make the edit based on a guess, so I left it as it was.